tests/integration-tests/tests/scheduler_plugin/test_scheduler_plugin.py

```diff
@@ -75,6 +76,7 @@ def test_scheduler_plugin_integration(
         another_instance=ANOTHER_INSTANCE_TYPE,
         user1=SCHEDULER_PLUGIN_USERS_LIST[0],
         user2=SCHEDULER_PLUGIN_USERS_LIST[1],
+        account_id="'" + account_id + "'",
```

Comment:
nit: you can pass account_id here without the quotes and in the yaml have `'{{account_id}}'`

Reply:
Good idea, changed, tested and it works